eng/build.yml

```diff
@@ -188,6 +259,13 @@ jobs:
             continueOnError: true
 
     - ${{ if notIn(parameters.agentOs, 'Windows_NT', 'Windows_NT_FullFramework', 'Windows_NT_TestAsTools', 'Windows_Perf_CI', 'Linux_Perf_CI', 'Windows_Perf_Helix', 'Windows_Perf_CI_FullFramework', 'Windows_Perf_Helix_Fullframework', 'Linux_Perf_Helix') }}:
+      - task: Bash@3
```

Comment:
Here's the problem, this needs to be conditional. Same w/ line 241. I'll push a fix